ring/util_test.go

```diff
@@ -257,10 +257,10 @@ func TestWaitRingStability_ShouldReturnErrorIfInstancesAddedAndMaxWaitingIsReach
 	defer close(done)
 
 	startTime := time.Now()
-	require.Equal(t, context.DeadlineExceeded, WaitRingStability(context.Background(), ring, Reporting, minStability, maxWaiting))
+	require.Error(t, context.DeadlineExceeded, WaitRingStability(context.Background(), ring, Reporting, minStability, maxWaiting))
```

Comment:
This currently tests that `context.DeadlineExceeded` is an error, not that `WaitRingStability` returned one.

Reply:
Btw, Same problem is in other places of this PR (eg. lines 280, 331)

Reply:
dang, fixed